sql/core/benchmarks/DateTimeRebaseBenchmark-jdk11-results.txt

```diff
@@ -6,49 +6,49 @@ OpenJDK 64-Bit Server VM 11.0.6+10-post-Ubuntu-1ubuntu118.04.1 on Linux 4.15.0-1
 Intel(R) Xeon(R) CPU E5-2670 v2 @ 2.50GHz
 Save dates to parquet:                    Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns)   Relative
 ------------------------------------------------------------------------------------------------------------------------
-after 1582, noop                                   9392           9392           0         10.6          93.9       1.0X
-before 1582, noop                                  9324           9324           0         10.7          93.2       1.0X
-after 1582, rebase off                            20975          20975           0          4.8         209.7       0.4X
-after 1582, rebase on                             20016          20016           0          5.0         200.2       0.5X
-before 1582, rebase off                           20088          20088           0          5.0         200.9       0.5X
-before 1582, rebase on                            20310          20310           0          4.9         203.1       0.5X
+after 1582, noop                                  18597          18597           0          5.4         186.0       1.0X
```

Comment:
@cloud-fan @HyukjinKwon Overhead of preparing input data depends on the system time zone. I opened this PR to have a base line for the optimisation https://github.com/apache/spark/pull/28119